conbench/templates/compare-entity.html

```diff
@@ -71,7 +64,13 @@
         <ul class="list-group">
           <li class="list-group-item active">Baseline</li>
           <li class="list-group-item" style="overflow-y: auto;">
-            <b>run</b>
+            <b>result ID</b>
```

Comment:
Imo, one of the more important properties of a result is its ID.

Reply:
As a user who hits our API a lot, I agree. I'm less sure that's true for more general users, but since our users are devs who certainly could hit the API if they wanted, I'm not opposed.

Reply:
Note that JP didn't switch out run ID for result ID; he just added result ID to the top of each column here.